x-pack/solutions/security/packages/expandable-flyout/README.md

```diff
@@ -2,53 +2,79 @@
 
 ## Purpose
 
-This package offers an expandable flyout UI component and a way to manage the data displayed in it. The component leverages the [EuiFlyout](https://github.com/elastic/eui/tree/main/src/components/flyout) from the EUI library.
+This package offers an expandable flyout UI component and a way to manage the data displayed in it. The component
```

Comment:
i noticed a lot of changes related to breaking sentencing into new line.. not sure if this is intended or a new auto-format tool?

Reply:
yeah I took care of it in my last commit and disable the auto formatting I recently enabled in Webstorm. Should be fixed now